### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by laura-a-n-n** [laura-a-n-n/tf-nlr#1]
Throughout the entire implementation process, I have found that the neural SDF model requires a learning rate that is 100 times smaller than what is stated in the paper. This may be related to some other training issue. That being said, using 1e-6 on S and 1e-4 on E seems to work well.
